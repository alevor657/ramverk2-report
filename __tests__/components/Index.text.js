// Index.react-test.js
import React from 'react';
import Index from '../../app/js/app/components/Index';
import renderer from 'react-test-renderer';

test('Index page renders', () => {
    const component = renderer.create(<Index/>);
    let tree = component.toJSON();

    expect(tree).toMatchSnapshot();
});
