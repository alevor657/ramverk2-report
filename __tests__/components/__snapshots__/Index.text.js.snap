// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Index page renders 1`] = `
<div
  className="container me-container"
>
  <h1>
    About me
  </h1>
  <div
    className="img-wrap"
  >
    <img
      className="me-img"
      src="test-file-stub"
    />
  </div>
  <div
    className="me-text"
  >
    <p>
      Hej, Jag heter Alexey och jag är 21 år gammal. Innan jag hade flyttat till Karlskrona så bodde jag med mina föräldrar i norra Stockholm, och innan dess så bodde jag i Moskva. Jag var alltid intreserad av teknik allmänt, men ännu mer av datorer och programmering.
    </p>
    <p>
      Jag tycker att det är spännande att skapa nya saker och automatisera dagliga rutiner. Jag brukar alltid vara den “händige” som fixar datorer/telefoner och så vidare. Sedan jag var en 14-åring så fick jag laga kompisars mobiler och datorer.
    </p>
    <p>
      Jag har även utvecklat 2 mindre webbplatser åt en företag, dessa kan man se här och här. Då hade jag använt mig av bootstrap, smoothscroll, ajax och ett par andra prylar. Nu så ser jag fram emot nya utmaningar och kunskaper!
    </p>
  </div>
</div>
`;
